### DISCUSSION TITLE
How  to display fan speed and ssd/mobo temperatures in Dashboard?

### DISCUSSION THREAD

**Discussion by lavrentius-dev** [webmin/webmin#2342] [Q&A] [answered]
First time Webmin user, Ubuntu server 24.04.1. Setting Webmin | Webmin Configuration | Background Status Collection shows that CPU, fan speed, and drive temperature collection is enabled. However, in the Dashboard only CPU temperatures are shown. Package **lm-sensors** is installed and works as expected with command **sensors** displaying temperatures from multiple devices. What am I missing?

**Comment by jcameron** [comment]
@iliajie any idea about this one? I think you worked on this code most recently ..

**Comment by iliaross** [comment, marked as answer]
@jcameron Yes, it should be implemented in the latest Webmin [development build](https://download.webmin.dev).
